### DISCUSSION TITLE
How to update to new snapshot without deleting current world progres and data

### DISCUSSION THREAD

**Discussion by HACKERPRO908** [itzg/docker-minecraft-server#3093] [General]
I have a physical server box thing hosting a Minecraft java server with docker but I want to update from 1.21.X to the new snapshot with all the creaking and the new biome without deleting my buildings and removing my inventory progress so how do I do it? 
SNAPSHOT the snapshot is 24W40A and future 1.22 updates

**Comment by itzg** [comment]
It'll depend how you created the container and if you used a volume mount for /data. If using a compose file it's as simple as setting VERSION to "SNAPSHOT" and applying with the command `docker compose up -d`. 

Take a backup before you do anything though since some of the snapshot world changes might be destructive.
